app/controllers/foods_controller.rb

```diff
@@ -0,0 +1,45 @@
+class FoodsController < ApplicationController
+  def index
+    @user = User.find(params[:user_id])
+    # @user = User.first
+    @foods = Food.find_by(user_id: params[:id])
+  end
+
```

Comment:
- [ ] to keep your code clean , kindly remove any commented code from your project 🙏